snuba/query/processors/__init__.py

```diff
@@ -0,0 +1,56 @@
+from abc import ABC, abstractmethod
```

Comment:
This entire file is just moved from query/query_processors.py. There should be no change.